pkg/docker/config/config.go

```diff
@@ -739,13 +752,27 @@ func stripScheme(url string) string {
 	return stripped
 }
 
+// normalizeRegistry converts the provided registry to contain only the hostname.
 func normalizeRegistry(registry string) string {
```

Comment:
The input is not a registry; and the output can be a host:port, not just a hostname.

(Yes, the function is already misnamed, but let’s not double down.)

Maybe
- `normalizeRegistry`+`stripRegistry` → `normalizeAuthFileKey(key, legacyFormat)` (I don’t think we need two different ones)
- `convertDockerHost` → `normalizeRegistry`

Reply:
Renamed the function and changed the docs.